### PR TITLE
19247 - Add Disability Calculator analytics.

### PR DESCRIPTION
**Pull request by tlei123**
## Description
[19247](/department-of-veterans-affairs/vets.gov-team/issues/19247) - Add Google Analytics events to Disability Combined Rating Calculator, as well as upcoming Combined ratings table’s accordion toggles.  See this [ticket comment](/department-of-veterans-affairs/vets.gov-team/issues/19247#issuecomment-520847648).

## Testing done
Using browser dev-tools, verified specified events were pushed to Google Analytics’ window.dataLayer.

## Acceptance criteria
- [ ] Event named “widget-disability-ratings-calculator-calculate” pushed to window.dataLayer, as seen in browser dev-tools, upon filling two ratings and clicking **Calculate** button.
NOTE: Another push to this branch is needed AFTER a [formation update](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/154) has been published, to bump fomation dependency, then the accordion-togglers can be tested here.

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/static-pages/subscribeAccordionEvents.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export default function subscribeAccordionEvents() {
+  document.body.addEventListener(
+    '@department-of-veterans-affairs/formation/accordion/button-clicked',
+    e => {
+      recordEvent({
+        event:
+          e.detail.toggle === 'expand'
+            ? 'nav-accordion-expand'
+            : 'nav-accordion-collapse',
+      });
```
Comment (ncksllvn) [inline src/applications/static-pages/subscribeAccordionEvents.js]: Picky, but maybe we could do `nav-accordion-${e.detail.toggle}`?

**Review by ncksllvn — APPROVED** [review]
LGTM! Are you also planning on doing the Formation bump in this PR to receive the updates in https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/154?

File: src/applications/static-pages/subscribeAccordionEvents.js (hunk 1 above)
Comment (tlei123) [reply src/applications/static-pages/subscribeAccordionEvents.js]: Update pushed.  Thx!

Comment (tlei123) [reply src/applications/static-pages/subscribeAccordionEvents.js]: Re. the tools repo PR154, the formation version-bump is already included.

Once approved, I'll merge and publish, then I'll push a formation bump here to match.

**Comment by tlei123** [comment]
Tools-repo PR154 is stuck on a Jenkins build-failure that is not replicable on local.  Added a comment there about how to fix that.

Once that PR has been approved, and formation update published.  I'll push the matching dependency bump up here so that this can pass the final Jenkins check(s), and then merge.

**Comment by tlei123** [comment]
Hi, @department-of-veterans-affairs/frontend-review-group !  This is now ready for your review/approval.

Formation dependency now bumped to match published version on NPM.

Thanks in advance!

**Comment by tlei123** [comment]
hi, @department-of-veterans-affairs/frontend-review-group !  This is now ready for your review.

I've just bump Formation dependency to match new published version on NPM \[from tools [PR154](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/154)].

Thanks in advance!

**Review by ncksllvn — APPROVED** [review]
LGTM 👍

**Review by rianfowler — APPROVED** [review]

**Pull request merged**